op-batcher/metrics/metrics.go

```diff
@@ -278,14 +297,16 @@ func (m *Metrics) RecordChannelClosed(id derive.ChannelID, numPendingBlocks int,
 }
 
 func (m *Metrics) RecordL2BlockInPendingQueue(block *types.Block) {
-	size := float64(estimateBatchSize(block))
-	m.pendingBlocksBytesTotal.Add(size)
-	m.pendingBlocksBytesCurrent.Add(size)
+	daSize, rawSize := estimateBatchSize(block)
+	m.pendingBlocksBytesTotal.Add(float64(rawSize))
+	m.pendingBlocksBytesCurrent.Add(float64(rawSize))
+	atomic.AddInt64(&m.pendingDABytes, int64(daSize))
 }
 
 func (m *Metrics) RecordL2BlockInChannel(block *types.Block) {
-	size := float64(estimateBatchSize(block))
-	m.pendingBlocksBytesCurrent.Add(-1 * size)
+	daSize, rawSize := estimateBatchSize(block)
```

Comment:
I'm wondering if we can use the current (and queued up) channel's compressed size(s) instead, which is more accurate and doesn't need any additional compression.

Reply:
Don't think so because these bytes we are counting are not already in a channel (the backlog).